backend/queries/volunteers.js

```diff
@@ -94,7 +94,7 @@ const addVolunteer = async (user, password) => {
     const volunteer = await db.one(insertQuery, user);
 
     const promises = [];
-    user.skills.map(skillId => promises.push(db.none('INSERT INTO volunteer_skills (volunteer_id, skill_id) VALUES ($1, $2)', [user.userId, skillId])));
+    user.skills.map(skillId => promises.push(db.none('INSERT INTO volunteer_skills (volunteer_id, skill_id) VALUES ($1, $2)', [volunteer.v_id, skillId])));
```

Comment:
same notes as older p.req

Reply:
I will switch to forEach, good catch
but as explained in the answer for the previous PR, I don't see the use of sending both data
we can discuss this during our meeting later one this evening